packages/client/src/views/SearchResult/SearchResult.tsx

```diff
@@ -86,18 +90,14 @@ const ToolTipContainer = styled.span`
   text-align: center;
 `
 
-export function getRejectionReasonDisplayValue(reason: string) {
-  switch (reason.toLowerCase()) {
-    case 'duplicate':
+export function getRejectionReasonDisplayValue(reason: RejectionReason) {
+  switch (reason) {
+    case RejectionReason.Duplicate:
       return rejectMessages.rejectionReasonDuplicate
-    case 'misspelling':
-      return rejectMessages.rejectionReasonMisspelling
-    case 'missing_supporting_doc':
-      return rejectMessages.rejectionReasonMissingSupportingDoc
```

Comment:
Can you please double-check these aren't used anywhere? I only found the strings from tests and I'm planning to remove the tests that refer to these.

Reply:
https://github.com/opencrvs/opencrvs-core/blob/5067150b285b40139a7ac6e92398685c240060ac/packages/client/src/views/SearchResult/SearchResult.test.tsx#L144-L145
Apparently at sometime it could have even been a comma separated string :D